apps/hash-frontend/src/pages/type-editor/blockprotocol-ontology-functions-hook.ts

```diff
@@ -32,28 +35,28 @@ import { useBlockProtocolUpdatePropertyType } from "../../components/hooks/block
 import { useIsReadonlyModeForApp } from "../../shared/readonly-mode";
 
 export type GraphMessageCallbacks = Omit<
-  EmbedderGraphMessageCallbacks,
-  | "getEntity"
-  | "createEntity"
-  | "aggregateEntities"
-  | "getEntityType"
-  | "createLink"
-  | "getLink"
-  | "updateLink"
-  | "deleteLink"
+  EmbedderGraphMessageCallbacks<true>,
   | "getLinkedAggregation"
   | "deleteEntity"
   | "deleteEntityType"
   | "createLinkedAggregation"
   | "updateLinkedAggregation"
   | "deleteLinkedAggregation"
-  // Replaced by new ontology callbacks
+  /** @todo-0.3 fix these inconsistencies */
```

Comment:
Very similar to [this thread](https://github.com/hashintel/hash/pull/2072/files#r1106017238)